src/nifgen/metadata/functions.py

```diff
@@ -2435,23 +2435,27 @@
             {
                 'direction': 'in',
                 'documentation': {
-                    'description': '\nSets the clock mode of the signal generator.\n\n****Defined Values****\n',
+                    'description': 'Trigger specifies the type of software trigger to send',
                     'table_body': [
                         [
-                            'NIFGEN_VAL_DIVIDE_DOWN'
+                            'NIFGEN_VAL_START_TRIGGER'
                         ],
                         [
-                            'NIFGEN_VAL_HIGH_RESOLUTION'
+                            'NIFGEN_VAL_SCRIPT_TRIGGER'
                         ],
-                        [
-                            'NIFGEN_VAL_AUTOMATIC'
-                        ]
-                    ]
+                    ],
+                    'table_header': [
+                        'Defined Values',
+                    ],
                 },
+                'enum': 'Trigger',
                 'name': 'trigger',
-                'type': 'ViInt32'
+                'type': 'ViInt32',
             },
             {
+                'documentation': {
+                    'description': 'Trigger ID specifies the Script Trigger to use for triggering.',
```

Comment:
I think we should provide more info on what the acceptable values are, like in [C reference](http://zone.ni.com/reference/en-XX/help/370524V-01/fgencref/cvinifgen_sendsoftwareedgetrigger/).

Reply:
Discussed offline and decided to leave it as is to keep it consistent with LV documentation.